docs/content/learn/tutorials/general/metadata.md

```diff
@@ -0,0 +1,828 @@
+
+A "Document Series Instance" is a discrete document instance that is part of a "Document Series". Using the above example, "Document 1 version 2 represented in OSCAL JSON" is a "Document Series Instance".
+
+In OSCAL we use a document identifier to identify that a document is a member of a "Document Series" and a document UUID to identify a "Document Series Instance".
+
+{{< tabs XML JSON YAML >}}
+{{% tab %}}
+{{< highlight xml "linenos=table" >}}
+<catalog uuid="c3da6d1d-c20c-4c7c-ae73-4010167a186b">
+  <metadata>
+    <title>Example OSCAL Document</title>
+    <published>2021-01-01T00:00:00-5:00</published>
+    <last-modified>2021-01-05T00:00:00-5:00</last-modified>
+    <version>1.0.0<version>
+    <oscal-version>1.0.4</oscal-version>
+    <document-id scheme="http://www.doi.org/">10.1000/182</document-id>
+  </metadata>
+  …
+</catalog>
+{{< /highlight >}}
+
+In the above example, OSCAL we add a `<document-id>` to track the "Document Series" identified by the [Digital Object Identifier](https://www.doi.org/driven_by_DOI.html) (DOI) System using the identifier `10.1000/182`.
+
+The DOI System is indicated using the `@scheme` attribute value `http://www.doi.org/`, which is standardized in OSCAL ([XML](/reference/latest/complete/xml-definitions/#/field/oscal-metadata/document-id/scheme), [JSON/YAML](/reference/latest/complete/json-definitions/#/field/oscal-metadata/document-id/scheme)). A schema is required to be a [URI](/reference/datatypes/#uri).
+{{% /tab %}}
+{{% tab %}}
+{{< highlight json "linenos=table" >}}
+{
+  "catalog": {
+    "uuid": "c3da6d1d-c20c-4c7c-ae73-4010167a186b"
+    "metadata": {
+      "title": "Example OSCAL Catalog",
+      "published": "2021-01-01T00:00:00-5:00",
+      "last-modified": "2021-01-05T00:00:00-5:00",
+      "version": "1.0.0",
+      "oscal-version": "1.0.4",
+      "document-ids": [
+        {
+          "scheme": "http://www.doi.org/",
+          "value": "10.1000/182"
+        }
+      ]
+    }
+  }
+}
+{{< /highlight >}}
+
+In the above example, OSCAL we add an object to the `document-ids` array to track the "Document Series" identified by the [Digital Object Identifier](https://www.doi.org/driven_by_DOI.html) (DOI) System using the identifier `10.1000/182`.
+
+The DOI System is indicated using the `scheme` property value `http://www.doi.org/`, which is standardized in OSCAL ([XML](/reference/latest/complete/xml-definitions/#/field/oscal-metadata/document-id/scheme), [JSON/YAML](/reference/latest/complete/json-definitions/#/field/oscal-metadata/document-id/scheme)). A schema is required to be a [URI](/reference/datatypes/#uri).
```

Comment:
```suggestion
The DOI System is indicated using the `scheme` property value `http://www.doi.org/`, which is standardized in OSCAL ([XML](/reference/latest/complete/xml-definitions/#/field/oscal-metadata/document-id/scheme), [JSON/YAML](/reference/latest/complete/json-definitions/#/field/oscal-metadata/document-id/scheme)). A scheme is required to be a [URI](/reference/datatypes/#uri).
```

Reply:
I am not sure about the URI requirement. What other schemes could we have for document identifiers and what would their schemes be?

For example, CMS identifiers or controlled document identifiers such as "SP 800-53rev5upd3" or "ANSI/NISO Z39.102-2017" (a controlled identifier assigned to a publication by an SDO). How should their respective ID schemes be identified?